docs/src/references/c8y-configuration-management.md

```diff
@@ -64,8 +91,38 @@ $ ls -l /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 -rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 ```
 
+The configuration plugin can also act as configuration proxy for child-devices.
+For that to work for a child device named `$CHILD_DEVICE_ID`,
+two files must be added under `/etc/tedge/operations/c8y/$CHILD_DEVICE_ID`
+in order to declare the associated capabilities to Cumulocity.
+
+These two files can be created using the `c8y_configuration_plugin --init` option and providing child names:
+
+```shell
+$ sudo c8y_configuration_plugin --init child-1 child-2
+
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+-rw-r--r-- 1 tedge tedge 95 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+  
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
+-rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
+```
+
 The `c8y_configuration_plugin` has to be run as a daemon on the device, the latter being connected to Cumulocity.
 
+On start of `tegde_mapper c8y` and on `/etc/tedge/operations/c8y` directory updates,
+one can observe on the MQTT bus of the thin-edge device
+the messages sent to Cumulocity to declare the capabilities of the main and child devices.
+Here, the capabilities to upload and download configuration files
+(possibly with other capabilities added independently):
+
+```shell
+$ tedge mqtt sub 'c8y/s/ds/#'
+[c8y/s/ds] 114,c8y_Restart,c8y_SoftwareList,c8y_UploadConfigFile,c8y_DownloadConfigFile
+[c8y/s/ds/child-1] 114,c8y_UploadConfigFile,c8y_DownloadConfigFile
+[c8y/s/ds/child-2] 114,c8y_UploadConfigFile,c8y_DownloadConfigFile
```

Comment:
`c8y/s/ds` is the topic which a device receives a smartrest message from Cumulocity. Isn't it `c8y/s/us`? I think these message direction is from thin-edge device to Cumulocity.

Reply:
I might be wrong. I will double-check and fix it.

Reply:
I was wrong => fixed